File: app/src/main/java/com/example/movies/ui/movie/view/MovieFragment.java
package com.example.movies.ui.movie.view;

import com.example.movies.core.base.BasePageableFragment;

/**
 * Created by Tarik on 16.11.2019.
 */

public class MovieFragment extends BasePageableFragment {

    @Override
    public void fetchRequest() {
        service.fetchMovies(pagination, this);
    }
}

File: app/src/main/java/com/example/movies/ui/movie/view/MovieFragment.kt
package com.example.movies.ui.movie.view

import com.example.movies.core.base.BasePageableFragment

/**
 * Created by Tarik on 16.11.2019.
 */

class MovieFragment : BasePageableFragment() {

    override fun fetchRequest() {
        service.fetchMovies(pagination, this)
    }
}
